pkg/apis/metal3/v1alpha1/zz_generated.openapi.go

```diff
@@ -1,4 +1,4 @@
-// +build !
+// +build !ignore_autogenerated
```

Comment:
Is this comment what is causing lint to start failing? Strange if it is... even stranger if not.

Reply:
I'm not sure. These zz_ files are autogenerated by operator-sdk. Now that my pull request is updating version of the operator-sdk something breaking the lint. I don't have any idea what might cause this lint to fail?